src/cache.js

```diff
@@ -0,0 +1,48 @@
+import _ from "underscore";
+
+function getKey(key) {
+  if (key.cid) return key.cid;
```

Comment:
for net new code I think we're agreeing we'd prefer not to do one line if / return statements right?  (I know this was a code move, but just wanted to check and wondering if we should tweak hound for this

Reply:
> for net new code I think we're agreeing we'd prefer not to do one line if / return statements right?

I think so. Are agreements are captured in invoca/style-guide.